README.md

```diff
@@ -154,10 +154,24 @@ We welcome pull requests from the community.
 To run tests on the source code, you'll need a running JFrog Artifactory Pro instance.
```

Comment:
Artifactory Pro for Artifactory tests
Xray for the Xray tests
Distribution and an Enterprise+ license for Distribution tests